theme.json

```diff
@@ -68,11 +68,28 @@
 		"spacing": {
 			"spacingScale": {
 				"operator": "+",
-				"increment": 0.5,
-				"steps": 9,
-				"mediumStep": 1.5,
+				"increment": 1,
+				"steps": 8,
+				"mediumStep": 3,
 				"unit": "rem"
 			},
+			"spacingSizes": [
+				{
+					"size": "max(1.25rem, 5vw)",
+					"slug": "35",
+					"name": "Flexi Small"
+				},
+				{
+					"size": "clamp(2rem, 8vw, calc(4 * var(--wp--style--block-gap)))",
+					"slug": "45",
```

Comment:
Is the `45` preset used anywhere? I only see `40` being used, so am curious why it's being added.

Reply:
No it's not currently being used anywhere, but this size was available with TT2 so I thought it would be good to include it, especially as it's a fluid preset.

Reply:
What's the reasoning behind using the `X5` instead of overriding core's `X0` values? (i.e. `35` instead of `30`).

Reply:
The reason I went with `X5` values was because they're fluid values, and this set them apart from the `X0` values.

As they're fluid, they don't necessarily fit between the other sizes, as these value names suggest. But I'm also not sure what else to call them!

Reply:
Could the X0 values be fluid instead?

Reply:
I think we'd need to manually specify each preset in the `spacingSizes` array to make each size fluid. Or is there currently a way to make each size in `spacingScale` fluid automatically?

Or maybe it would be enough to specify each TT2 fluid size (small, medium, and large) in the `spacingSizes` array as fluid presets, and not have any added by `spacingScale`?